interface/forms/note/common.php

```diff
@@ -0,0 +1,127 @@
+ * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+
+require_once(__DIR__ . "/../../globals.php");
+require_once("$srcdir/api.inc.php");
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Core\Header;
+
+formHeader("Form: note");
+$returnurl = 'encounter_top.php';
+$provider_results = sqlQuery("select fname, lname from users where username=?", array($_SESSION["authUser"]));
+
+/* name of this form */
+$form_name = "note";
+
+// get the record from the database
+if ($_GET['id'] != "") {
+    $obj = formFetch("form_" . $form_name, $_GET["id"]);
+}
+
+?>
+<html>
+<head>
+    <title><?php echo xlt('School/Work Note'); ?></title>
+    <?php Header::setupHeader('datetime-picker'); ?>
+
+    <script>
+        // required for textbox date verification
+        var mypcc = <?php echo js_escape($GLOBALS['phone_country_code']); ?>;
+
+        $(function () {
+            $('.datepicker').datetimepicker({
+                <?php $datetimepicker_timepicker = false; ?>
+                <?php $datetimepicker_showseconds = false; ?>
+                <?php $datetimepicker_formatInput = true; ?>
+                <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
+                <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
+            });
+        });
+
+        function PrintForm() {
+            newwin = window.open(<?php echo js_escape($rootdir . "/forms/" . $form_name . "/print.php?id=" . urlencode($_GET["id"])); ?>,"mywin");
+        }
+
+    </script>
+</head>
+<body>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-12 mt-5">
+            <h1><?php echo xlt('School/Work Note'); ?></h1>
+            <?php echo text(date("F d, Y", time())); ?>
+        </div>
+    </div>
+    <form method=post action="<?php echo $rootdir . "/forms/" . $form_name . "/save.php?mode=update&id=" . attr_url($_GET["id"]);?>" name="my_form" id="my_form">
+        <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+        <div class="m-3">
+            <select name="note_type" class="form-control">
+                option value="WORK NOTE" <?php if ($obj['note_type'] == "WORK NOTE") {
+                    echo " SELECTED";
+                } ?>><?php echo xlt('WORK NOTE'); ?></option>
+                <option value="SCHOOL NOTE" <?php if ($obj['note_type'] == "SCHOOL NOTE") {
+                    echo " SELECTED";
+                } ?>><?php echo xlt('SCHOOL NOTE'); ?></option><
+            </select>
+        </div>
+        <div class="m-4">
+            <b><?php echo xlt('MESSAGE:'); ?></b>
+            <textarea class="form-control" name="message" id="message" cols ="67" rows="4"><?php echo text($obj["message"]);?></textarea>
```

Comment:
in places where outputting something from $obj, ensure to do following to avoid php warnings:
```
text($obj["message"] ?? '')
```